Automattic/jetpack
# Protect: Add paid UI using Scan API data source

#### Description
* This PR introduces the separation between free and paid components using the Scan API data source in the Protect admin UI based on the Scan in Protect integration designs. 

#### Changes proposed in this Pull Request:

* Adds a Free and a Paid Accordion component
* Adds a Free and a Paid List component
* Adds a ThreatSeverityBadge component
* Displays the correct components based on the presence of the Scan module

#### Other information:

#### Jetpack product discussion


* 1201069996155224-as-1202738501526880
* p1664305449611359-slack-C029WFNV69M 

#### Does this pull request change what data or activity we track or use?


* No

#### Testing instructions:





* Checkout this branch
* Start up your Jurassic Tube
* Activate Protect without an upgrade
* Proceed through the Free installation process and allow the initial status report to generate
* Compare against the free designs and ensure all functionality remains 
* Upgrade your site to Scan, Security Bundle, or Complete
* Activate Scan from My Jetpack, approve a user connection, and allow the initial Scan to process
* Return to the Protect tab, compare the status report against the new paid designs and ensure all functionality remains 
   * At the time of publishing this, only the Ignore threats button was functional - Fix all threats (#),  Scan now, Fix threats, and manual scan buttons are all currently placeholders
